webdriver_wharf/db.py

```diff
@@ -10,6 +10,8 @@
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy.ext.declarative import declarative_base
 
+from app import vnc_enabled, httpd_enabled
```

Comment:
How about a new file specifically for environment variables? Doesn't solve the larger problems but for purposes of this PR it would do the trick of resolving the import graph issue.